src/screens/CreatePost/index.js

```diff
@@ -18,7 +18,6 @@ import {
   Text,
   View
 } from 'react-native';
-import {Image} from 'react-native-compressor';
 import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
 import {openSettings} from 'react-native-permissions';
 import {showMessage} from 'react-native-flash-message';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `Image` from 'react-native-compressor' has been removed. Ensure that this does not break any functionality where the `Image` component was used previously.
```diff
- import {Image} from 'react-native-compressor';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->